### PR TITLE
Dra 327 dynamic injection openapi

### PR DESCRIPTION
**Pull request by VictorHarbo**

### CODE REVIEW

File: src/main/openapi/ds-storage-openapi_v1.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ paths:
           required: true
           schema:
             type: string
-            example: ${example_origin}
+            example: ${config:allowed_origins[0].name}
```
Comment (tokee) [inline src/main/openapi/ds-storage-openapi_v1.yaml:69]: Could this (an all the other similar usages) be an enum instead of a single value?

**Review by tokee — CHANGES_REQUESTED** [review]
Minor request for changelog change + wish for enums.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
-## Changed
-Added two new fields to record: kalturareferenceid and kalturainternalid. The kaltura internal id is required by the frontend for thumbnails and streaming. 
-New service method to update the kalturaId for a record. Both create new record and update record will set the kalturareferenceid.
+### Added
+- Support for dynamically updating values in OpenAPI spec through internal JIRA issue [DRA-139](https://kb-dk.atlassian.net/browse/DRA-139).
+
+### Changed
+- Added two new fields to record: kalturareferenceid and kalturainternalid. The kaltura internal id is required by the frontend for thumbnails and streaming. 
+- New service method to update the kalturaId for a record. Both create new record and update record will set the kalturareferenceid.
```
[the remark this answers is not in the document]
Comment (VictorHarbo) [reply CHANGELOG.md:14]: Nice catch. Of course!

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,9 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## Changed
-Added two new fields to record: kalturareferenceid and kalturainternalid. The kaltura internal id is required by the frontend for thumbnails and streaming. 
-New service method to update the kalturaId for a record. Both create new record and update record will set the kalturareferenceid.
+### Added
+- Support for dynamically updating values in OpenAPI spec through internal JIRA issue [DRA-139](https://kb-dk.atlassian.net/browse/DRA-139).
```
[the remark this answers is not in the document]
Comment (VictorHarbo) [reply CHANGELOG.md:10]: I get your point. Is the new version any better? Not sure how we best link jiras in a pretty way

[the remark this answers is not in the document]
Comment (tokee) [reply CHANGELOG.md]: I think the links are fine with the simple issue-ID as tekst. It is pretty standard to have links to issues, so there aren't even a need for stating that they are JIRA issues.

**Pull request merged**